_includes/overall-schedule.md

```diff
@@ -23,5 +23,29 @@ Organizers will inform participants of the week schedule by email.
 {%- capture date %}{% if c.type != "Mentor-Mentee" %}{{ c.date }} {% if c.time %}([{{ c.time }} European Time](https://arewemeetingyet.com/Berlin/{{ c.date | date: "%Y-%m-%d" }}/{{ c.time }}/{{ cohort }}%20{{ c.type }}%20Call%20(Week%20{{ w[0] }}))){% endif %}{% endif %}{% endcapture %}
 | {{ w-desc }} | [{{ c.type }}](/{{ cohort }}#{{ c.type | downcase | remove: "(" | remove: ")" | remove: "@" | remove: ":" | remove: "," | replace: " ", "-" | remove: "&"  }}-calls) | {{ date }} | [**{{ c.title }}**](/{{ cohort }}/schedule#week-{{ w[0] }}) | {% if c.agenda %}{{ c.agenda }}{% endif %} |
 {%- assign w-desc = "" %}
+{%- endfor %} 
 {%- endfor %}
-{%- endfor %}
+
+<script type="application/ld+json" >
+{
+  "@context": "https://schema.org",
+  "@id": "{{ site.url }}/{{ cohort }}/schedule",
+  "@type": "Course",
+  "dct:conformsTo": "https://bioschemas.org/profiles/Course/0.9-DRAFT-2020_12_08",
+  "description":  "{{site.description}}",
+    "hasCourseInstance": [
+    {
+      "@context": "https://schema.org",
+      "@type": "CourseInstance",
+      "dct:conformsTo": "https://bioschemas.org/profiles/CourseInstance/0.8-DRAFT-2020_10_06",
+      "courseMode": "{%- for w in schedule.weeks %} {%- capture w-desc %}**Week {{ w[0] }}** (start. {{ w[1].start }}){% endcapture %} {%- for c in w[1].calls %}{%- capture date %} {% if c.type !='Mentors-Mentee'%} {{ c.date }} {% if c.time %} ([{{ c.time }} European Time] (https://arewemeetingyet.com/Berlin/{{ c.date | date: '%Y-%m-%d' }}/{{ c.time }}/{{ cohort }}%20{{ c.type }}%20Call%20(Week%20{{ w[0] }}))) {% endif %}{% endif %}{% endcapture %} | {{ w-desc }} | [{{ c.type }}](/{{ cohort }}#{{ c.type | downcase | remove: '(' remove: | remove:')'| remove: '@' | remove: ':' | remove: ',' | replace: ' ', '-' | remove: '&'}}-calls) | {{ date }} | [**{{ c.title }}**] (/{{ cohort }}/schedule#week-{{ w[0] }}) | {% if c.agenda %}{{ c.agenda }}{% endif %} |{%- assign w-desc = '' %} {%- endfor %} {%- endfor %}"
```

Comment:
Given the documentation, `courseMode`: 

> The medium, means or pace of delivery of the course instance or the mode of study, either as a text label (e.g. “online”, “onsite”, “hybrid” or “blended”; “synchronous” or “asynchronous”; “full-time” or “part-time”) or as a URL reference to a term from a controlled vocabulary (e.g. https://ceds.ed.gov/element/001311#Asynchronous ).

So here it should be `["online", "synchronous"]`

Reply:
The information you currently put in `courseMode` should be split with other properties:
- [`startDate`](http://schema.org/startDate) using `{{ c.date }}`
- [`endDate`](http://schema.org/endDate) using `{{ c.date }}`
- [`duration`](http://schema.org/duration) using `{{ c.date }}`
- [`name`](http://schema.org/name) using `{{ c.title }}`